### PR TITLE
Updating Travis Migration Docs

### PR DESCRIPTION
**Pull request by PK1312**
# Description
I moved the "On Using Containers" section to be higher up, so it's one of the first things customers moving over from Travis and checking our docs see- since all Travis builds are based on specifying a language and CircleCI builds are done in docker containers, and that's information customers need before they can really take any other steps forward, I think having it earlier up in the table of contents makes sense. I've also included a table translating terms from Travis's yml configuration into the equivalent CircleCI terms with links to the relevant sections for quick reference.

# Reasons
I'm making these changes to help make it easier for Travis users to migrate to CircleCI.

### CODE REVIEW

File: jekyll/_cci2/migrating-from-travis.md [hunk 1]
```diff
@@ -33,6 +33,31 @@ configuration will live in a `.travis.yml` file in the root of your repository.
...
+| after_script:     | [run:](https://circleci.com/docs/2.0/configuration-reference/#run)                        | Use the run: step to specify post-execution commands                                              |
+| deploy:           | [deploy:](https://circleci.com/docs/2.0/configuration-reference/#deploy)                     | Use the deploy: step to deploy build artifacts                                                    |
+| env:              | [environment:](https://circleci.com/docs/2.0/configuration-reference/#environment)                | Use the environment: element to specify environment variables                                     |
+| matrix:           | [workflows:](https://circleci.com/docs/2.0/configuration-reference/#workflows)                  | Workflows are used on CircleCI to orchestrate multiple jobs                                       |
+| stage:            | [requires:](https://circleci.com/docs/2.0/configuration-reference/#requires)                   | Use the requires: element to explicitly require any job dependencies and control parallel builds. |
+
```
Comment (michelle-luna) [inline jekyll/_cci2/migrating-from-travis.md]: actually need to add this line at end of the table for formatting:

{: class="table table-striped"}

Comment (PK1312) [reply jekyll/_cci2/migrating-from-travis.md]: Done 👍

**Comment by PK1312** [comment]
@michelle-luna It doesn't look like this working- the "{: class="table table-striped"}" is not displaying in plaintext in the documentation 😱 are you sure it's required to work? if so, did I add it in the wrong place?

**Comment by michelle-luna** [comment]
@PK1312 thanks for checking, it needs a line break before it.

**Pull request merged**